compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/window/PopupTest.kt

```diff
@@ -554,4 +588,37 @@ class PopupTest {
         onNodeWithTag("box1").assertPositionInRootIsEqualTo(80.dp, 80.dp)
         onNodeWithTag("box2").assertPositionInRootIsEqualTo((-30).dp, (-30).dp)
     }
+
+    // https://github.com/JetBrains/compose-multiplatform-core/pull/847
+    @Test
+    fun popupBoundsWithPlatformInsets() = runSkikoComposeUiTest(
+        size = Size(200f, 200f)
+    ) {
+        setPlatformInsets(PlatformInsets(50.dp, 10.dp, 5.dp, 5.dp))
```

Comment:
```suggestion
        setPlatformInsets(PlatformInsets(left = 5.dp, top = 50.dp, right = 5.dp, bottom = 10.dp))
```

Let's also change the default order of `PlatformInsets` to `left, top, right, bottom`, to be consistent with the other Compose classes (WindowInsets, PaddingValues)

Reply:
Good catch. It was copy-paste from `IOSInsets`